master/internal/user/postgres_users.go

```diff
@@ -43,7 +42,6 @@ WHERE u.id = ?`
 // UserByUsername looks up a user by name in the database.
 func UserByUsername(username string) (*model.User, error) {
 	var user model.User
-	username = strings.ToLower(username)
```

Comment:
im not sure the intention of making `username` lower case, but postgres `unique` is case sensitive, so i removed this line